drivers/bridge/setup_ip_tables.go

```diff
@@ -114,6 +114,15 @@ func (n *bridgeNetwork) setupIPTables(config *networkConfiguration, i *bridgeInt
 
 	n.portMapper.SetIptablesChain(filterChain, n.getNetworkBridgeName())
 
+	if config.Internal {
+		if err = setupInternalNetworkRules(config.BridgeName, maskedAddrv4, true); err != nil {
+			return fmt.Errorf("Failed to Setup IP tables: %s", err.Error())
+		}
+		n.registerIptCleanFunc(func() error {
+			return setupInternalNetworkRules(config.BridgeName, maskedAddrv4, false)
+		})
+	}
+
 	return nil
 }
```

Comment:
I think in this function we should skip the nat chain programming and all the other calls to `setupIPTablesInternal()` if `config.Internal == true`